.github/workflows/assign_issues_to_project.yaml

```diff
@@ -0,0 +1,24 @@
+name: Scan Pull Request Activity
+
+on:
+  schedule:
+    - cron:  '0 1 * * *'
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+  issues: read
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: scan pull request activity
+        uses: skarlso/caretaker@v0.5.0
+        with:
+          command: scan
+          owner: skarlso
+          repo: test
+          token: ${{ secrets.CARETAKER_TOKEN }}
```

Comment:
I need a token lalalala. 🎤

Reply:
ocm-bot token?

Reply:
do you know what's the secret called? :)